Ginger/Ginger/Activities/ActivitiesGroupPage.xaml.cs

```diff
@@ -76,7 +79,131 @@ public ActivitiesGroupPage(ActivitiesGroup activitiesGroup, BusinessFlow parentB
+                {
+                    selectedActivitiesList.Add(actID.IdentifiedActivity);
+                }
+            }
+
+            if (selectedActivitiesList != null && selectedActivitiesList.Count > 0)
+            {
+                foreach (Activity sharedActivity in selectedActivitiesList)
+                {
+                    if (senderGrid != null)
+                    {
+                        Activity newInstance = sharedActivity.CreateInstance(true) as Activity;
+                        mActivitiesGroup.AddActivityToGroup(newInstance);
+                    }
+                    else
+                    {
+                        mActivitiesGroup.RemoveActivityFromGroup(sharedActivity);
+                    }
+                }
+            }
+        }
+
+        void ShiftItem(bool ShiftUp)
+        {
+            ActivityIdentifiers selectedActivity = grdGroupedActivities.CurrentItem as ActivityIdentifiers;
+
+            if (selectedActivity != null)
+            {
+                int currentIndex = grdGroupedActivities.DataSourceList.IndexOf(selectedActivity);
+                int lastIndex = 0;
+                int newIndex = -1;
+                if (!ShiftUp)
+                {
+                    lastIndex = grdGroupedActivities.DataSourceList.Count - 1;
+                    newIndex = currentIndex + 1;
+                }
+                else
+                {
+                    newIndex = currentIndex - 1;
+                }
+
+                if (currentIndex == lastIndex)
+                {
+                    return;
+                }
+                else
+                {
+                    grdGroupedActivities.DataSourceList.Move(currentIndex, newIndex);
+                }
+            }
+        }
+
+        private void MoveDown_Click(object sender, EventArgs e)
+        {
+            ShiftItem(false);
+        }
+
+        private void MoveUp_Click(object sender, EventArgs e)
+        {
+            ShiftItem(true);
+        }
+
+        private void BtnDown_Click(object sender, RoutedEventArgs e)
+        {
+            //ActivityIdentifiers selectedActivity = grdGroupedActivities.CurrentItem as ActivityIdentifiers;
+
+            //if(selectedActivity != null)
+            //{
+            //    int currentIndex = grdGroupedActivities.DataSourceList.IndexOf(selectedActivity);
+            //    int lastIndex = grdGroupedActivities.DataSourceList.Count - 1;
+
+            //    if(currentIndex == lastIndex)
+            //    {
+            //        return;
+            //    }
+            //    else
+            //    {
+            //        grdGroupedActivities.DataSourceList.Move(currentIndex, currentIndex + 1);
+            //    }
+            //}
+            //throw new NotImplementedException();
+        }
+
+        private void BtnUp_Click(object sender, RoutedEventArgs e)
+        {
+            //throw new NotImplementedException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4582993)